pegasus/sites.v3/code.org/views/minecraft_2022_promo_older.haml

```diff
@@ -0,0 +1,36 @@
+:ruby
+  timecraft_image = "/images/mc/fit-720x620/mc_mee_half_timecraft.png"
+  timecraft_title = I18n.t(:hoc2021_mc_mee_title)
+  timecraft_description = I18n.t(:hoc2022_mc_mee_timecraft_description)
+  timecraft_link = "https://aka.ms/HourofCode2021"
+
+  villages_image = "/images/mc/fit-720x620/mc_mee_half_villages.jpg"
+  villages_title = I18n.t(:hoc2021_mc_mee_villages_title)
+  villages_description = I18n.t(:hoc2021_mc_mee_villages_description)
+  villages_link = "https://aka.ms/hourofcode2020"
```

Comment:
It looks like both this url and the `/HourofCode2021` url are meant to be from previous years? The links themselves look right, so just double checking.

Reply:
Yep! Those are the correct links.